src/Uno.UWP/System/Display/DisplayRequest.cs

```diff
@@ -0,0 +1,50 @@
+﻿#if __ANDROID__ || __IOS__
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Windows.System.Display
+{
+	public partial class DisplayRequest
+	{
+		private static object _syncLock = new object();
+		private static int _globalRequestCount = 0;
+
+		private int _instanceRequestCount = 0;
+
+		public void RequestActive()
+		{
+			lock (_syncLock)
+			{
+				_instanceRequestCount++;
+				_globalRequestCount++;
+				if (_globalRequestCount == 1)
+				{
+					//first global request, activate screen lock
+					ActivateScreenLock();
+				}
+			}
+		}
+
+		public void RequestRelease()
+		{
+			lock (_syncLock)
+			{
+				if (_instanceRequestCount == 0)
+					throw new InvalidOperationException("No active display request to release");
+				_instanceRequestCount--;
+				_globalRequestCount--;
+				if ( _globalRequestCount == 0)
```

Comment:
Formatting

Reply:
Subtle, was confused where the problem is the first time I saw it :-D .